app/Actions/Service/StartService.php

```
@@ -15,6 +18,26 @@ class StartService
     public function handle(Service $service, bool $pullLatestImages = false, bool $stopBeforeStart = false)
     {
         $service->parse();
+        $edgeRoutingWarnings = [];
+        $edgePortForwardWarnings = [];
+        try {
+            $edgeRoutingWarnings = app(EdgeProxyRemoteRouteService::class)->syncService($service);
+        } catch (\Throwable $exception) {
+            Log::warning('Failed to sync edge proxy route for service start.', [
+                'service_uuid' => $service->uuid,
+                'error' => $exception->getMessage(),
+            ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'handle'. (undefined)

(StaticAccess)